CHANGELOG/CHANGELOG-1.6.md

```diff
@@ -1,5 +1,22 @@
 # Changelog 1.6
 
+## 1.6.1 - 2022-05-11
+
+### Added
+
+- Add [documentation](https://github.com/antrea-io/antrea/blob/release-1.6/docs/security.md#protecting-your-cluster-against-privilege-escalations) for the Antrea Agent RBAC permissions and how to restrict them using Gatekeeper/OPA. ([#3694](https://github.com/antrea-io/antrea/pull/3694),[@antoninbas])
+
+### Fixed
+
+- Clean up stale routes installed by AntreaProxy when ProxyAll is disabled. ([#3465](https://github.com/antrea-io/antrea/pull/3465), [@hongliangl])
+- Fix export/import of Services with named ports when using the Antrea Multi-cluster feature. ([#3561](https://github.com/antrea-io/antrea/pull/3561), [@luolanzone])
+- Fix handling of the "reject" packets generated by the Antrea Agent to avoid infinite looping. ([#3569](https://github.com/antrea-io/antrea/pull/3569), [@GraysonWu])
+- Fix DNS resolution error of Antrea Agent on AKS by using `ClusterFirst` dnsPolicy. ([#3701](https://github.com/antrea-io/antrea/pull/3701), [@tnqn])
+- Fix tolerations for Pods running on control-plane for Kubernetes >= 1.24. ([#3731](https://github.com/antrea-io/antrea/pull/3731), [@xliuxu])
+- Reduce permissions of Antrea Agent ServiceAccount. ([#3691](https://github.com/antrea-io/antrea/pull/3691), [@xliuxu])
+- [Windows] Non-Service traffic generated by kubelet could bypass NetworkPolicies when ProxyAll is enabled. ([#3510](https://github.com/antrea-io/antrea/pull/3510), [@hongliangl])
```

Comment:
Actually non-service traffic generated by kubelet could bypass NetworkPolicies with or without the PR.
Maybe `[Windows] Ensure that Service traffic does not bypass NetworkPolicies when ProxyAll is enabled.`

Reply:
thanks!